docs/enroute/Melbourne Centre/ASP.md

```diff
@@ -76,5 +78,32 @@ Any aircraft that will enter AS ADC airspace, and not landing at YBAS, must be H
     <span class="hotline">**ASP** -> **AS ADC**</span>: "via AS, JDS"  
     <span class="hotline">**AS ADC** -> **ASP**</span>: "JDS, A060"
 
+### WR ADC
+
+Departures from YPWR in to WRA Class E will be coordinated when ready for departure.
+
+!!! example
+    <span class="hotline">**WR ADC** -> **WRA**</span>: "Next, ASY404"  
+    <span class="hotline">**WRA** -> **WR ADC**</span>: "ASY404, Unrestricted"  
+    <span class="hotline">**WR ADC** -> **WRA**</span>: "Unrestricted, ASY404" 
+
+The Standard Assignable level from **WR ADC** to ASP is the lower of `F130` or the `RFL`, any other level must be prior coordinated.
+
+#### Arrivals
+YPWR arrivals shall be heads-up coordinated to **WR ADC** from WRA prior to **5 mins** from the boundary.
+
+!!! example
+    <span class="hotline">**WRA** -> **WR ADC**</span>: "Via WR, ASY219”  
```

Comment:
Same as the format requested on WR ADC page.

WRA -> WR ADC: "ASY219, what level can I assign?"
WR ADC -> WRA: "ASY219, A023"
WRA -> WR ADC: "A023, ASY219"

WRA: "ASY219, When ready, leave and re-enter controlled airspace tracking DCT WR, on descent to A023, no reported IFR Traffic."
ASY219: "When ready, leave and re-enter controlled airspace tracking DCT WR, on descent to A023, ASY219"